app/cells/select/select_cell.rb

```diff
@@ -0,0 +1,48 @@
+class SelectCell < FormCellBase
+  attribute :form, mandatory: true, description: 'A form object.'
```

Comment:
Shouldn't attributes have some kind of type? How do I know if `classes` has to be an array or a space-separated string?